target/linux/lantiq/xrx200/config-5.4

```diff
@@ -50,7 +50,11 @@ CONFIG_MTD_UBI_BLOCK=y
 # CONFIG_MTD_UBI_FASTMAP is not set
 # CONFIG_MTD_UBI_GLUEBI is not set
 CONFIG_MTD_UBI_WL_THRESHOLD=4096
+CONFIG_NET_DSA=y
```

Comment:
Please deactivate ```CONFIG_SWCONFIG``` this is not needed any more.

Reply:
IN addition you also have to deatcivte these settings:
```
# CONFIG_PSB6970_PHY is not set
# CONFIG_RTL8366_SMI is not set
# CONFIG_SWCONFIG is not set
```
Please also refresh the configuration, this DSA driver also activates PHYLINK.
```
make kernel_oldconfig CONFIG_TARGET=subtarget
```